CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+- **CUMULUS-3934**
+  - Updated `ecs_cluster_instance_allow_ssh` resource to best security practices
+  - Restricted CIDR range of above resource to the VPC to address vulnerability
```

Comment:
We should discuss further in a DM, but this should probably be called out more/as a breaking change.

Reply:
Updated the release notes to be a bit more descriptive in the latest commits.